### DISCUSSION TITLE
React Router subPaths

### DISCUSSION THREAD

**Discussion by RealMarvn** [remix-run/react-router#8787] [Q&A]
Got a question to React Router DOM v6.
```jsx
function App() {
    return (
        <React.Suspense fallback={Home}>
            <ThemeProvider theme={theme}>
                <Routes>
                    <Route index element={<Standard element={<Home/>}/>}/>
                    <Route path='/dashboard' element={<Standard element={<Dashboard/>}/>}/>

                    <Route path="/control/:id" element={<Control/>}>
                        <Route path='welcomer' element={<Welcomer/>}/>
                        <Route path='tempchannel' element={<TempChannel/>}/>
                        <Route path='notifications' element={<Notifications/>}/>
                    </Route>
                </Routes>
            </ThemeProvider>
        </React.Suspense>
    );
}
}``` 
This is how my tree looks like currently but I want to do it that way:
```
domain.de/dashboard -> dashboard
domain.de/dashboard/:ID -> ControlPanel
domain.de/dashboard/:ID/welcomer -> ControlPanel element = Welcomer
domain.de/dashboard/:ID/tempchannel -> ControlPanel element = TempChannel
```

I dont really understand how react router dom is working in v6 but i saw it does have this stuff like subPaths...How can I do this?
I now have the routes I want with Control. But there is a problem...Images won't load correctly...They are not found somehow. And /dashboard is a completly other component than control. I cant just add an Outlet to /dashboard because i dont want to show any of those components on Dashboard.tsx in subRoutes .... I only want to show the Outlet in routes like /welcomer ..

**Comment by Moshyfawn** [comment]
You just nest `Route`s inside each other to construct your layout.

See [this codesanbox](https://codesandbox.io/s/wonderful-curran-xzw9g8?file=%2Fsrc%2FApp.tsx)

**Reply by RealMarvn** [reply]
Yeah but  the problem is that the Dashboard Component on my side is a HEADER , MAIN , FOOTER and my Control Panel does only have a Sidebar and a Component. The Header stays in your example...But on the Control Panel component , there must be no header or footer. You know what I mean?

**Reply by RealMarvn** [reply]
This is the Dashboard
![image](https://user-images.githubusercontent.com/67916561/162620079-7daac489-9aa9-4f27-be0c-8c08053b0396.png)

**Reply by RealMarvn** [reply]
And this is the Control Panel....There has to be no Footer or Header.....This is looking completly different. Thats why Outlet is so confusing to me...Because If I use it like you did , the header would stay
![image](https://user-images.githubusercontent.com/67916561/162620112-e10513f9-4424-4b57-8bb1-bcacc2681d7e.png)

**Reply by Moshyfawn** [reply]
You've gotta play around with nested `Route`s. I've updated [the codesanbox](https://codesandbox.io/s/wonderful-curran-xzw9g8?file=%2Fsrc%2FApp.tsx)

**Reply by RealMarvn** [reply]
Ah yeah now I understand!
```function App() {
    return (
        <React.Suspense fallback={Home}>
            <ThemeProvider theme={theme}>
                <Routes>
                    <Route index element={<Standard element={<Home/>}/>}/>
                    <Route path='/dashboard' element={<Standard element={<Dashboard/>}/>}/>
                    <Route path="/dashboard/:id" element={<ControlPanel/>}>
                        <Route index element={<Notifications/>}/>
                        <Route path='join-manager' element={<JoinManager/>}/>
                        <Route path='temp-channel' element={<TempChannel/>}/>
                        <Route path='notifications' element={<Notifications/>}/>
                    </Route>
                </Routes>
            </ThemeProvider>
        </React.Suspense>
    );
}```

**Reply by RealMarvn** [reply]
This works! Great! Thanks again for your help :D.....Do you maby have Discord? I got some questions regarding TS itself. Would it be okay if I ask you sometimes?

**Reply by RealMarvn** [reply]
Ive got another problem!
Images arent loading correctly:
![image](https://user-images.githubusercontent.com/67916561/162620700-ceef826e-7c97-48fe-911e-1dc2725457ab.png)
This is not working if I start routing....Hmmmm
